docs/plugins/subsonicplaylist.rst

```diff
@@ -0,0 +1,33 @@
+Subsonic Playlist Plugin
+========================
+
+The ``subsonicplaylist`` plugin allows to import playlists from a subsonic server.
+This is done by retrieving the track infos from the subsonic server, searching
+them in the beets library and adding the playlist names to the
+`subsonic_playlist` tag of the found items. The content of the tag has the format:
+
+    subsonic_playlist: ";first playlist;second playlist"
+
+To get all items in a playlist use the query `;playlist name;`.
+
+Command Line Usage
+------------------
+
+To use the ``subsonicplaylist`` plugin, enable it in your configuration (see
+:ref:`using-plugins`). Then use it by invoking the ``subsonicplaylist`` command.
```

Comment:
I think it would be great to include a minimal config, with required values, here. Something like this… "Next, configure the plugin to connect to your Subsonic server, like this:"

```yaml
subsonicplaylist:
    base_url: http://subsonic.example.com
    username: me
    password: seekrit
```

"Then you can use it by invoking the `subsonicplaylist` command."